servlet-security/pom.xml

```diff
@@ -257,19 +226,18 @@
                     <plugin>
                         <groupId>org.wildfly.plugins</groupId>
                         <artifactId>wildfly-maven-plugin</artifactId>
-                        <version>${version.wildfly.maven.plugin}</version>
                         <configuration>
                             <feature-packs>
                                 <feature-pack>
                                     <location>org.wildfly:wildfly-galleon-pack:${version.server}</location>
                                 </feature-pack>
                                 <feature-pack>
-                                    <location>org.wildfly.cloud:wildfly-cloud-galleon-pack:${version.cloud.fp}</location>
+                                    <location>org.wildfly.cloud:wildfly-cloud-galleon-pack:${version.pack.cloud}</location>
                                 </feature-pack>
                                 <feature-pack>
```

Comment:
since you going to do changes, please take the opportunity to change the feature pack specification similar to cloud one: 
`<location>org.wildfly:wildfly-datasources-galleon-pack:${version.pack.datasources}</location>`